nixos/modules/services/web-apps/glitchtip.nix

```diff
@@ -224,26 +248,41 @@ in
         };
       in
       {
-        glitchtip = commonService // {
+        glitchtip = {
           description = "GlitchTip";
+          inherit environment;
+          bindsTo = [ "glitchtip-worker.service" ];
```

Comment:
why is that? shouldn't it (in theory) be possible to start the services independently?